gnd/src/main/res/layout/map_container_frag.xml

```diff
@@ -32,90 +32,96 @@
     android:layout_width="match_parent"
     android:layout_height="match_parent">
 
+    <!-- Feature holder for map implementation -->
+    <FrameLayout
+      android:id="@+id/map"
+      android:layout_width="match_parent"
+      android:layout_height="match_parent" />
 
-    <RelativeLayout
-      android:id="@+id/map_layout"
+    <FrameLayout
       android:layout_width="match_parent"
       android:layout_height="match_parent"
-      android:layout_gravity="center"
-      android:gravity="center"
-      app:layout_behavior="com.google.android.gnd.ui.home.mapcontainer.MapLayoutBehavior">
+      android:fitsSystemWindows="true">
```

Comment:
I could never get this to work; nicely done!

Reply:
I have read in Romain Guy's blog that flat hierarchy views are better because they consume less resources and are rendered quickly. So, that was one of my motivation too.